### PR TITLE
41-edit issue description

### LINKED ISSUE #41: Edit issue description
## Summary

The user selects an issue and edits the description of the same issue.

## Acceptance criteria

```gherkin
Scenario:  Select an issue to edit
Given the user is viewing issue I-x
When the user selects to edit the issue 
Then the user sees all the options to edit the issue

Scenario:  Edit the description
Given the user is viewing the options to edit issue I-x
When the user selects to edit the description 
And enters the new valid description
Then the user sees a notification 

Scenario:  Edit the description
Given the user is viewing the options to edit issue I-x
When the user selects to edit the description
Then the user is asked to enter a new description


```

### PR DESCRIPTION
**Pull request by ebarylko**
## Summary

Added a menu, `editDescription`, for allowing the user to edit the description of an issue.

Added a menu, `editAnticipatedRelease`, for allowing the user to edit the anticipated release of an issue.

### CODE REVIEW

**Review by angineer02 — APPROVED** [review]
looks great i will make sure to add db to my imports as well :))

File: src/main/kotlin/anttracker/Release.kt [hunk 1]
```diff
@@ -31,6 +32,12 @@ class Release(
     var releaseDate: LocalDate,
 )
 
+data class ReleasePage(
```
Comment (tyrus-tracey) [inline src/main/kotlin/anttracker/Release.kt]: Is this an old page implementation?

File: src/main/kotlin/anttracker/Release.kt [hunk 2]
```diff
@@ -31,6 +32,12 @@ class Release(
     var releaseDate: LocalDate,
 )
```
Comment (tyrus-tracey) [inline src/main/kotlin/anttracker/Release.kt:33]: We should axe this and other classes/IVCs replaced by dao entities.

File: src/main/kotlin/anttracker/issues/Issues.kt [hunk 3]
```diff
@@ -53,6 +131,17 @@ private fun selectIssueToViewMenu(rows: RowToIssuePage) =
         }
     }
 
+private fun viewIssueMenu(rows: RowToIssuePage) =
+    screenWithMenu {
+        title("View issue")
+        content { t ->
+            t.prompt(
```
Comment (tyrus-tracey) [inline src/main/kotlin/anttracker/issues/Issues.kt:137]: Probably once we got most of the modules working together we should do a sweep to see what UI differs from what's in the user manual.

File: src/main/kotlin/anttracker/Release.kt (hunk 1 above)
Comment (ebarylko) [reply src/main/kotlin/anttracker/Release.kt]: Yes, I believe so.

File: src/main/kotlin/anttracker/issues/Issues.kt (hunk 3 above)
Comment (ebarylko) [reply src/main/kotlin/anttracker/issues/Issues.kt:137]: As we go, we should check. Otherwise, it will be a lot of potential work.

**Pull request merged**